web/api/v1/api_test.go

```diff
@@ -3227,6 +3242,8 @@ func testEndpoints(t *testing.T, api *API, tr *testTargetRetriever, es storage.E
 						ctx = route.WithParam(ctx, p, v)
 					}
 
+					model.NameValidationScheme = test.nameValidationScheme
```

Comment:
unfortunately this will cause flakiness because t.Run can (does?) run things concurrently. In other similar tests I had to serialize the test so that the value stays consistent.

Reply:
Hey @ywwg. Does that sound right to you? It would be good to get this closed off

Reply:
ugh sorry for losing track of this PR.

because model.NameValidationScheme is a global value, one goroutine could set it to one value while another routine expects it to be another value, leading to flakiness.  If you remove the `t.Run` declaration, the tests will run serially and they will work fine.

Reply:
AFAIK, tests [shouldn't run concurrently](https://go.dev/play/p/o47jzzukckF) unless [there's a `t.Parallell()` call](https://go.dev/play/p/uyTmgU14RCm) (why there's no race detector on go playground?)

Reply:
oh interesting.  I'll see if it is actually flaky